8data.tex

```diff
@@ -26,13 +26,13 @@
 \subsection[Units Of Measure]{\protect\hyperlink{UnitsOfMeasure}{Units of Measure}}
 The normal units of measure are as follows:
 \begin{itemize}
-	\item Catch biomass - metric tons	
-	\item Body weight - kilograms	
+	\item Catch biomass - metric tons (mt)
```

Comment:
Why put the abbreviations here and then not on the metric tons below or use the abbreviation once it is defined?